test/integration/vaultauthmethods_integration_test.go

```diff
@@ -256,3 +379,34 @@ func TestVaultAuthMethods(t *testing.T) {
 		})
 	}
 }
+
+func alwaysRun() (bool, string) { return true, "" }
+
+// checks whether or not to run the aws tests
+func runAWS() (bool, string) {
+	if v := os.Getenv(envSkipAWS); v == "true" {
+		return false, envSkipAWS + " is set to 'true'"
+	}
+	return true, ""
+}
+
+// checks whether or not to run the static creds test
+func runAWSStaticCreds() (bool, string) {
+	if run, why := runAWS(); !run {
+		return run, why
+	}
+	if v := os.Getenv(envSkipAWSStaticCreds); v == "true" {
+		return false, envSkipAWSStaticCreds + " is set to 'true'"
+	}
+	tfVars := []string{
+		"TF_VAR_aws_access_key_id",
+		"TF_VAR_aws_secret_access_key",
+		"TF_VAR_aws_static_creds_role",
+	}
+	for _, tfv := range tfVars {
```

Comment:
I wonder if we should fail the test since the required env vars are not set.

Reply:
Sure, that makes sense.